app/views/projects/details.html.erb

```diff
@@ -0,0 +1,117 @@
+    <% if @project_eligible_to_edit && current_user.superuser %>
+      <%= link_to "Edit", edit_project_path(@project.id), id:"revisible-heading", class: "btn btn-primary btn-sm" %>
+    <% end %>
+  </h2>
+  <dl>
+    <% if !@data_sponsor.nil? %>
+      <dt>Data Sponsor</dt>
+      <dd id="data_sponsor">
+        <%= @data_sponsor.display_name_safe %>
+      </dd>
+    <% end %>
+    <% if !@data_manager.nil? %>
+      <dt>Data Manager</dt>
+      <dd id="data_manager">
+        <%= @data_manager.display_name_safe %>
+      </dd>
+    <% end %>
+
+    <dt>Data Users</dt>
+    <% if @data_users.present? %>
+      <dd>
+        <%= @data_user_names %>
+      </dd>
+    <% else %>
+      <dd>
+        <p>None</p>
+      </dd>
+    <% end %>
+  </dl>
+</div>
+
+<div class="details">
+  <h2>Project Description
+    <%# Per ticket #1114 only superusers have edit access %>
+    <% if @project_eligible_to_edit && current_user.superuser %>
+      <%= link_to "Edit", edit_project_path(@project.id), id:"revisible-heading", class: "btn btn-primary btn-sm" %>
+    <% end %>
+  </h2>
+  <dl>
+    <dt>Departments</dt>
+    <% if @departments.empty? %>
+      <dd><strong class="px-0">None</strong></dd>
+    <% else %>
+      <dd>
+        <% @project.project.metadata_json["departments"].each do |department_code| %>
+          <% Affiliation.all.each do |hash| %>
+            <% if hash[:code] == department_code %>
+            <%= hash[:name] %>
+            <% end %>
+          <% end %>
+        <% end %>
+      </dd>
+    <% end %>
+    <dt>Project Directory</dt>
+    <dd><%= @project.project_directory %></dd>
+    <dt>Title</dt>
+    <dd><%= @project.title %></dd>
+    <dt>Description:<dt/> <dd><%= @project.description %></dd>
+  </dl>
+</div>
+
+<div class="details">
+  <%= if @project.pending?
+    render partial: "pending_details"
+  else
+    render partial: "approved_details"
+  end %>
+</div>
+
+<div class="details">
+  <h2> Provenance </h2>
+  <p class="provenance-header"> Submission </p>
+  <ul class>
+    <% @provenance_events.each do |event| %>
+      <li class="provenance"> <%= event.event_details %>, <%=event.created_at.to_time.in_time_zone("America/New_York").iso8601%> </li>
+    <% end %>
+  </ul>
+</div>
+
+<div class="details">
+  <% if current_user.eligible_sysadmin? %>
+    <% if @project.pending? %>
+      <%= link_to "Approve Project", project_approve_path(@project.id), class: "btn btn-primary btn-sm" %>
+      <%= link_to " View Create Script", "#", class: "bi bi-code btn btn-secondary btn-sm", id: "create-script-btn" %>
+    <% end %>
+  <% end %>
+</div>
+
+<!-- this div is populated by the AJAX below -->
+<div id="create-script-text"></div>
+</show>
+
+<script type="module">
+  // Make the AJAX call to fetch the aterm script
+  showCreateScript('<%= project_create_script_path(format: "json") %>');
+  projectStyle('<%= @project_session %>');
+  projectTab("<%= project_path %>", "<%= project_details_path %>");
```

Comment:
@JaymeeH I changed the JavaScript to pass the URLs that Rails generates so that the JavaScript in `projectTab()` does not have to guess what's the correct URLs for contents and details.